newIDE/app/src/SceneEditor/index.js

```diff
@@ -106,6 +106,7 @@ type State = {|
   variablesEditedInstance: ?gdInitialInstance,
   variablesEditedObject: ?gdObject,
   selectedObjectNames: Array<string>,
+  addFirstInstancePosition: Array<number>,
```

Comment:
Prefer using `?Array<number>` to allow null. You can also try `?[number, number]`.